### PR TITLE
Fix IPv6 PD auto routing

### PR DESCRIPTION
**Pull request by marjohn56**
Add the missing exec() calls.

### CODE REVIEW

File: src/sbin/prefixes.php [hunk 1]
```diff
@@ -60,7 +60,8 @@
 // echo "add routes\n";
 if (count($routes) > 0) {
     foreach ($routes as $address => $prefix) {
-        echo "/sbin/route change -inet6 {$prefix} {$address}\n";
```
Comment (fichtner) [inline src/sbin/prefixes.php]: hmmm, I think this is it, "route change" was a custom patch a long time ago in a project far far away

**Comment by fichtner** [comment]
Can you try c62f01f993b2ab on master instead? We should still fix the command pipe in abit, this is fishy... but first things first. Very nice catch!

**Comment by marjohn56** [comment]
Appears to work. :)

Applied the patch, manually deleted the route on the primary, reset the secondary WAN and the lease is back...

**Comment by fichtner** [comment]
Thank you, I'll weave in your original changes with the next commit. Very very good find. I owe you a pint. 👍

**Pull request closed without merging**